src/components/RecentList/styles.scss

```
@@ -0,0 +1,44 @@
+@import '../../../assets/scss/variables/index.scss';
+@import '../../../assets/scss/mixins/index.scss';
+.recent-list{
+    margin-top: 40px;
+    margin-left: $menuWidth;
+    @include padding( 0 50px);
```

Mấy cái nhỏ như này phải kiểm tra kỹ trước khi push em nhé
```suggestion
@include padding(0 50px);
```